src/services

```diff
@@ -53,3 +53,5 @@ p({"class":"foam.nanos.boot.NSpec", "name":"professorDAO", "lazy":true, "serve":
 p({"class":"foam.nanos.boot.NSpec", "name":"studentDAO", "lazy":true, "serve":true, "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setJournaled(true).setJournalName(\"students\").setOf(foam.nanos.demo.relationship.Student.getOwnClassInfo()).build();", "client":"{\"of\":\"foam.nanos.demo.relationship.Student\", \"remoteListenerSupport\": true}"})
 p({"class":"foam.nanos.boot.NSpec", "name":"courseDAO", "lazy":true, "serve":true, "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setJournaled(true).setJournalName(\"courses\").setGuid(true).setOf(foam.nanos.demo.relationship.Course.getOwnClassInfo()).build();", "client":"{\"of\":\"foam.nanos.demo.relationship.Course\", \"remoteListenerSupport\": true}"})
 p({"class":"foam.nanos.boot.NSpec", "name":"studentCourseJunctionDAO", "lazy":true, "serve":true, "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setJournaled(true).setJournalName(\"studentCourseJunction\").setOf(foam.nanos.demo.relationship.StudentCourseJunction.getOwnClassInfo()).build();", "client":"{\"of\":\"foam.nanos.demo.relationship.StudentCourseJunction\", \"remoteListenerSupport\": true}"})
+
```

Comment:
Put in its own services file under support.

Reply:
ignore comment